com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Fixed
 - Fixed client throwing an exception if it has messages in the outbound queue when processing the NetworkEvent.Disconnect event and is using UTP. (#1884)
 
+## [Unreleased]
+
+### Fixed
```

Comment:
I merged `develop` back into this PR and now we have 2 "Unreleased" and "Fixed" sections.
Do you mind cleaning up this file a bit and move things around before merging this PR?